test/hyperdome_common_test/test_available_port.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+"""
+Hyperdome
+
+Copyright (C) 2019-2020 Skyelar Craver <scravers@protonmail.com>
+                   and Steven Pitts <makusu2@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from hyperdome.common.common import get_available_port
+from hypothesis import given, assume
+import hypothesis.strategies as st
+import pytest
+import socket
+
+MAX_PORT = 65535
+MIN_PORT = 0
+
+MAX_USED_PORT = 62125
+MIN_USED_PORT = 62105
+
+MAX_UNUSED_PORT = 63125
+MIN_UNUSED_PORT = 63105
+
+
+@pytest.fixture(scope="session")
+def bound_ports():
+    sockets = []
+    for i in range(MIN_USED_PORT, MAX_USED_PORT):
```

Comment:
Should this be `(MIN_USED_PORT, MAX_USED_PORT+1)`? Not that it really matters

Reply:
yeah it, it needs to be inclusive of MAX, it's probably clearer if it's in the for.